lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -28,6 +29,8 @@ class HtmlViewEmbedder {
   /// The [HtmlViewEmbedder] singleton.
   static HtmlViewEmbedder instance = HtmlViewEmbedder._();
 
+  DomElement get skiaSceneHost => (renderer as CanvasKitRenderer).sceneHost!;
```

Comment:
For code that assumes CanvasKit we could have a dedicated static getter that's typed as `CanvasKitRenderer` (perhaps `CanvasKitRenderer.instance`). Then a type cast won't be necessary.